Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -1725,6 +1814,8 @@ private static T FindImmediateChild<T>(HierarchyNode parent)
                         return "e,env,environment: p,package: a,admin";
                     case PythonConstants.GenerateRequirementsTxt:
                         return "e,env,environment:";
+                    case PythonConstants.ProcessRequirementsTxt:
+                        return "path";
```

Comment:
Yeah that works fine, as I'm not using that parser. For some reason I always imagined there was a way to get parameter help from command window, I guess there isn't and that explains why I can't find it in the docs!